.github/workflows/my-ci-pipeline.yml

```diff
@@ -29,4 +29,10 @@ jobs:
           password: ${{ secrets.DOCKER_API_KEY }}
    - uses: actions/checkout@v2 
    - run: docker build --target production --tag aexalt/todo-app:latest .
-   - run: docker push aexalt/todo-app:latest
\ No newline at end of file
+   - run: docker push aexalt/todo-app:latest
+   - uses: actions/checkout@v2
+   - uses: akhileshns/heroku-deploy@v3.12.12
+     with:
+          heroku_api_key: ${{secrets.HEROKU_API_KEY}}
+          heroku_app_name: alt-devops-p8
+          heroku_email: andy.lee-tuffnell@lv.co.uk
```

Comment:
I'm a bit suspicious that this doesn't have the `useDocker` argument, and [the latest pipeline run that I can see attempting this failed](https://github.com/aexalt/DevOps-Course-Starter/actions/runs/2957055700)!

Switching that flag on should help, but be warned I believe this action doesn't have an option for specifying the Dockerfile "target", so you'll probably need to also move your `production` stage to be the last one defined in your `Dockerfile`